src/components/NodeList.js

```diff
@@ -18,55 +19,100 @@ const EnhancedNode = DragSource(selectable(Node));
 /**
   * Renders a list of Node.
   */
-const NodeList = ({
-  nodes,
-  nodeColor,
-  label,
-  selected,
-  onSelect,
-  itemType,
-  isOver,
-  willAccept,
-  meta,
-}) => {
-  const isSource = !!find(nodes, ['uid', get(meta, 'uid', null)]);
-
-  const classNames = cx(
-    'node-list',
-    { 'node-list--hover': !isSource && willAccept && isOver },
-    { 'node-list--drag': !isSource && willAccept }, // TODO: rename class
-  );
-
-  return (
-    <StaggeredTransitionGroup
-      className={classNames}
-      component="div"
-      delay={animation.duration.fast * 0.2}
-      duration={animation.duration.slow}
-      start={animation.duration.slow + 3}
-      transitionName="node-list--transition"
-      transitionLeave={false}
-    >
-      { isOver && willAccept &&
-        <Node key="placeholder" placeholder />
```

Comment:
If we don't have a placeholder node, do we need a different/stronger highlight effect on hover to differentiate between hover and dragging?

Reply:
Yes good point, it's yet to be determined so I've added it as an issue.

Reply:
I think we will probably just want to visually increase the intensity of the effect for a valid drop target when hovering.